### ISSUE TITLE
Source code?

### ISSUE THREAD

**Issue by K4sum1** [weolar/xpchrome#1]
The entire point of GitHub is to organize and publish source code easily. You're kinda expected to have source code on GitHub.

**Comment by Zero3K** [comment]
Please do it ASAP.

**Comment by Zero3K** [comment]
I guess you cannot because it is "proprietary", you are "selling it to companies", and it is "illegal" to do so because it is "under a NDA".

**Comment by Zero3K** [comment]
Yeah, the source code cannot be released since it is actually a closed source program.

**Comment by weolar** [comment]
Actually, I am also considering open-sourcing it. XPChrome is a paid product in China, but I plan to offer it to everyone in the form of open source, allowing free usage for self-compilation, while the binary version will be available for a fee.

**Comment by andika207** [comment]
> XPChrome is a paid product in China

pricing ? lifetime software license or do you charge for each new release ?

**Comment by diderius6** [comment]
Hi,
how to enable WebGL in this nice browser
Dietmar

**Comment by qwertyopus** [comment]
WebGL just works of the box. Yep, it's Disabled on chrome://gpu, but it's works. WebGL games work.

**Comment by qwertyopus** [comment]
On Windows XP WebGL working with libegl.dll and libglesv2.dll from another Chinese browser: TSBrowser (from Chromium 75 version) and with D3DCompiler_47.dll on BlackWingCat's msvcrt.dw7 Wrapper. Maybe work and with libgl files from another Chromium-based browser. I haven't checked them all.

**Comment by ghost** [comment]
Does it also enable hardware acceleration?

**Comment by qwertyopus** [comment]
Chromium-based Browsers not support HWA under XP from 39-version! But can work with SwiftShader.
